applications/TrilinosApplication/CMakeLists.txt

```diff
@@ -96,8 +96,8 @@ kratos_python_install(${INSTALL_PYTHON_USING_LINKS} ${CMAKE_CURRENT_SOURCE_DIR}/
 get_filename_component (CURRENT_DIR_NAME ${CMAKE_CURRENT_SOURCE_DIR} NAME)
 kratos_python_install_directory(${INSTALL_PYTHON_USING_LINKS} ${CMAKE_CURRENT_SOURCE_DIR}/python_scripts KratosMultiphysics/${CURRENT_DIR_NAME} )
 
-# TODO: Move this to the new system. Not sure how it will work on windows but being trillinos should give no problem.
-message("Trilinos libs to be installed" ${TRILINOS_LIBRARIES})
+# TODO: Move this to the new system. Not sure how it will work on windows but being Trilinos should give no problem.
+message(STATUS "Trilinos libs to be installed: " ${TRILINOS_LIBRARIES})
```

Comment:
Could it be that this change turned into a super verbose output?

Reply:
Shouldn't be, in any case STATUS is in the same level as no mode output :S

Reply:
It is basically the same with a minor decorator

Reply:
The point is that now the `Trilinos libs to be installed: ` is super verbose in my machine (maybe I didn't note it before).

Reply:
I just added STATUS, is not that different. You can check to remove STATUS locally. If that's the reason I will revert.